make/bundle-dist

```diff
@@ -24,18 +26,19 @@ ARCHIVE_ROOT="${GIT_ROOT}/output/scf-${stemcell_os}-${APP_VERSION}"
 echo Packaging, taking "${APP_VERSION}" ...
 
 # Assembling the pieces ...
-mkdir -p ${tmp_dir}/kube ${tmp_dir}/helm
+find "${tmp_dir}"
```

Comment:
This change seems to indicate the the tmpdir is expected to be present and that the coming commands create the subdirs ?

Reply:
Or, in light of the overall comment, no subdirs anymore at all ?

Reply:
`mkdir -d` creates the `tmpdir` (in a way that should avoid racing with other processes).  So, yes. (And line 33 does indeed create the subdirs; because it's `mkdir -p`, `tmpdir` technically doesn't even need to exist yet.)